airflow/providers/google/cloud/operators/compute.py

```diff
@@ -161,9 +163,10 @@ def __init__(
             self._field_validator = GcpBodyFieldValidator(
                 GCE_INSTANCE_TEMPLATE_VALIDATION_PATCH_SPECIFICATION, api_version=api_version
             )
+        resource_id = self.resource_id  # this line for validate-operators-init pre-commit
```

Comment:
It’s likely better to change all `self.resource_id` assignments to use this local variable, and pass it to `super().__init__`.